negotiation/contract.negotiation.protocol.md

```diff
@@ -39,6 +39,21 @@ The CN state machine is represented in the following diagram. Note that transiti
+|----------------------------|--------|-------------------------------------------|--------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| INIT                       | C      | ContractRequestMessage                    | REQUESTED    | The consumer requests a contract, without previous direct interaction with provider, e.g. through the usage of a central cataloging system, which provides all necessary information about the provider, to directly request a data contract.                                                                                            |
+| INIT                       | P      | ContractOfferMessage                      | OFFERED      | The provider sends a contract offer directly to a consumer without previous interaction, e.g. because the provider knows about desired data (e.g., consumer placed a search for data request in a catalog) or consumer/provider are in an existing business relationship and provider wants to ship information to its business partner. |
+| REQUESTED                  | P      | ContractOfferMessage                      | OFFERED      | The provider sends an offering for a requested resource.                                                                                                                                                                                                                                                                                 |
+| REQUESTED                  | P      | ContractAgreementMessage                  | AGREED       | The provider directly sends a contract agreement as no terms must be agreed to, e.g. in case publicly data shall be shared without any policies. Be aware that implementation of such transitions must comply with local laws in application.                                                                                            |
+| OFFERED                    | C      | ContractRequestMessage                    | REQUESTED    | The consumer declines the offered contract and might sent a counter offer to the provider.                                                                                                                                                                                                                                               |
```

Comment:
```suggestion
| OFFERED                    | C      | ContractRequestMessage                    | REQUESTED    | The consumer declines the offered contract by sending a (new) contract request (to be seen as counter offer) to the provider.                                                                                                                                                                                                                                               |
```

Reply:
```suggestion
| OFFERED | C | ContractRequestMessage | REQUESTED | The consumer declines the offered contract by sending a new contract request to the provider. |
```
- Technically it will always be a new contract request 
- "counter offer" is not defined. 

For curiosity (maybe discuss next week): How could a consumer define other parameters inside the ContractRequestMessage? WIthin the `odrl:Offer`?